src/models/tools/table/table-content.ts

```diff
@@ -34,6 +34,25 @@ export function getTableContent(target: IAnyStateTreeNode, tileId: string): Tabl
   return content && content as TableContentModelType;
 }
 
+interface IGetTableContentHeight {
+  dataRows: number;
+  rowHeight?: number;
+  readOnly?: boolean;
+  hasExpressions?: boolean;
+  padding?: number;
+}
```

Comment:
Probably want a line break between these.

Reply:
Since the interface is defining the API for the function, it's essentially part of the function definition, so I kept them close intentionally.